### PR TITLE
Added description of compressed classification models

### PR DESCRIPTION
**Pull request by ljaljushkin**

### CODE REVIEW

**Comment by ljaljushkin** [comment]
@AlexKoff88

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by kchechil** [comment]
Jenkins: test this please

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| Accuracy top-1 (ImageNet) |         78.36% |
+
+## Performance
+
+## Input
+
+Image, shape - `1,299,299,3`, format is `B,H,W,C` where:
```
Comment (eaidova) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: Converted modesl has `B,C,H,W` input format, (onnx as I know too)

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# inceptionv3-int8-onnx-0001
+
+## Use Case and High-Level Description
+
+This is the Inception v3 model that is designed to perform image classification. The model has been pretrained on the 
+ImageNet image database and then symmetrically quantized to INT8 fixed-point precision using Neural Network Compression 
+Framework (NNCF).  
+
+The model input is a blob that consists of a single image of "1x299x299x3" in BGR order.
```
Comment (eaidova) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: please change shape in the description also

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 11.469 |
+| MParams           | 23.817 |
+| Source framework  | PyTorch    |
```
Comment (eizamaliev) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]:
```suggestion
| Source framework  | PyTorch\*    |
```

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## Performance
+
+## Input
+
+Image, shape - `1,3,299,299`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
```
Comment (eizamaliev) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: Please, align list as it is above (`B,C,H,W`)

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# inceptionv3-int8-onnx-0001
```
Comment (eizamaliev) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: Add `Legal information` part (see other model's description for [example](https://github.com/opencv/open_model_zoo/blob/develop/models/intel/action-recognition-0001-decoder/description/action-recognition-0001-decoder.md))

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 11.469 |
+| MParams           | 23.817 |
+| Source framework  | PyTorch\*    |
+
+## Accuracy
+
+The quality metrics calculated on ImageNet validation dataset is 78.36% accuracy top-1.
```
Comment (eaidova) [inline models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: I think this line is redundant, all necessary information listed in the table below

File: models/intel/index.md [hunk 7]
```diff
@@ -187,12 +187,21 @@ Deep Learning compressed models
...
 | [inceptionv3-int8-sparse-v1-tf-0001](./inceptionv3-int8-sparse-v1-tf-0001/description/inceptionv3-int8-sparse-v1-tf-0001.md)                   | 11.469               | 23.819     |
 | [inceptionv3-int8-sparse-v2-tf-0001](./inceptionv3-int8-sparse-v2-tf-0001/description/inceptionv3-int8-sparse-v2-tf-0001.md)                   | 11.469               | 23.819     |
 | [mobilenetv2-int8-tf-0001](./mobilenetv2-int8-tf-0001/description/mobilenetv2-int8-tf-0001.md)                                                 | 0.615                | 3.489      |
 | [mobilenetv2-int8-sparse-v1-tf-0001](./mobilenetv2-int8-sparse-v1-tf-0001/description/mobilenetv2-int8-sparse-v1-tf-0001.md)                   | 0.615                | 3.489      |
 | [mobilenetv2-int8-sparse-v2-tf-0001](./mobilenetv2-int8-sparse-v2-tf-0001/description/mobilenetv2-int8-sparse-v2-tf-0001.md)                   | 0.615                | 3.489      |
+| [resnet50v1-int8-onnx-0001](./resnet50v1-int8-onnx-0001/description/resnet50v1-int8-onnx-0001.md)                                              | 8.216                | 25.53      |
```
Comment (IRDonch) [inline models/intel/index.md]: Please keep these in sorted order.

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md (hunk 5 above)
Comment (IRDonch) [reply models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: Also, please update `tools/downloader/license.txt`.

**Comment by AlexKoff88** [comment]
All models are postponed to the next release. Don't merge!

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md (hunk 5 above)
Comment (ljaljushkin) [reply models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: @yssemaev could you please help licenses?

**Comment by ljaljushkin** [comment]
> All models are postponed to the next release. Don't merge!

The situation has been changed. Now it's target for the upcoming release, please review

File: models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md (hunk 5 above)
Comment (ljaljushkin) [reply models/intel/inceptionv3-int8-onnx-0001/description/inceptionv3-int8-onnx-0001.md]: added as Apache License v2

**Comment by ljaljushkin** [comment]
all discussions are resolved, please review and merge if it's OK

**Comment by ljaljushkin** [comment]
@vladimir-dudnik applied naming conventions for classification quantized models

File: models/intel/index.md [hunk 8]
```diff
@@ -188,12 +188,21 @@ Deep Learning compressed models
 | [resnet-50-int8-tf-0001](./resnet-50-int8-tf-0001/description/resnet-50-int8-tf-0001.md)                                                       | 6.996                | 25.530     |
 | [resnet-50-int8-sparse-v1-tf-0001](./resnet-50-int8-sparse-v1-tf-0001/description/resnet-50-int8-sparse-v1-tf-0001.md)                         | 6.996                | 25.530     |
 | [resnet-50-int8-sparse-v2-tf-0001](./resnet-50-int8-sparse-v2-tf-0001/description/resnet-50-int8-sparse-v2-tf-0001.md)                         | 6.996                | 25.530     |
+| [resnet50v1-int8-onnx-0001](./resnet50v1-int8-onnx-0001/description/resnet50v1-int8-onnx-0001.md)                                              | 8.216                | 25.53      |
+| [resnetv1-101-int8-tf-0001](./resnetv1-101-int8-tf-0001/description/resnetv1-101-int8-tf-0001.md)                                              | 14.441               | 44.496     |
+| [resnet50-int8-sparse-v2-onnx-0001](./resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md)                      | 8.216                | 25.53      |
```
Comment (IRDonch) [inline models/intel/index.md]: Is there a difference between ResNet 50 and ResNet 50 v1? If not, then I think you should just remove "v1" from the names.

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/inceptionv3-int8-sparse-v2-onnx-0001/description/inceptionv3-int8-sparse-v2-onnx-0001.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+# inceptionv3-int8-sparse-v2-onnx-0001
+
+## Use Case and High-Level Description
+
+This is the Inception v3 model that is designed to perform image classification.
```
Comment (eizamaliev) [inline models/intel/inceptionv3-int8-sparse-v2-onnx-0001/description/inceptionv3-int8-sparse-v2-onnx-0001.md]: If original model exists in OMZ/public, please add link to it.

File: models/intel/resnet101-int8-tf-0001/description/resnet101-int8-tf-0001.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 14.441         |
+| MParams           | 44.496        |
+| Source framework  | TensorFlow    |
+
+## Accuracy
+
+The quality metrics calculated on ImageNet validation dataset is 74.5% accuracy top-1.
```
Comment (eizamaliev) [inline models/intel/resnet101-int8-tf-0001/description/resnet101-int8-tf-0001.md]: Duplication of table content, please remove (except, reference to dataset)

File: models/intel/resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
+# resnet50-int8-sparse-v2-onnx-0001
```
Comment (eizamaliev) [inline models/intel/resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md]: What does `v2` mean? Is this information crucial? What is the difference with `v1` in the model below?

File: models/intel/inceptionv3-int8-sparse-v2-onnx-0001/description/inceptionv3-int8-sparse-v2-onnx-0001.md (hunk 9 above)
Comment (vladimir-dudnik) [reply models/intel/inceptionv3-int8-sparse-v2-onnx-0001/description/inceptionv3-int8-sparse-v2-onnx-0001.md]: @eizamaliev link to original model not required there

File: models/intel/resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md (hunk 11 above)
Comment (vladimir-dudnik) [reply models/intel/resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md]: @eizamaliev v2 is version of sparsfication algorithm, do not mix it with version of topology

File: models/intel/index.md (hunk 8 above)
Comment (vladimir-dudnik) [reply models/intel/index.md]: naming was fixed

File: tools/downloader/license.txt [hunk 12]
```diff
@@ -3875,11 +3879,16 @@ License terms:
 * resnet-50-int8-sparse-v1-tf-0001
 * resnet-50-int8-sparse-v2-tf-0001
 * resnet-50-int8-tf-0001
+* resnet50-int8-sparse-v2-onnx-0001
+* resnet50v1-int8-onnx-0001
```
[the remark this answers is not in the document]
Comment (vladimir-dudnik) [reply tools/downloader/license.txt]: I fix that by myself

File: models/intel/resnet101-int8-tf-0001/description/resnet101-int8-tf-0001.md (hunk 10 above)
Comment (vladimir-dudnik) [reply models/intel/resnet101-int8-tf-0001/description/resnet101-int8-tf-0001.md]: @eizamaliev this is consistent description with already released quantized/sparsified models inceptionv3-int8-xxx, mobilenetv2-int8-xxx, resnet-50-int8-xxx.

**Pull request merged**